src/main/webapp/new/app/menu/game-page/game-page.component.ts

```diff
@@ -40,6 +46,18 @@ export class GamePageComponent implements OnInit, OnDestroy {
             .then(game => {
                 this.game = game;
 
+                if (this.game.isPlaying()) {
+                    this.game
+                        .getCurrentPlayer(this._authUser.get()).availableActions
+                        .onUpdate(actions => {
+                            actions.forEach(action => {
+                                if (action.notify) {
+                                    this._notification.notifyGlobal(action.notifyMessage, action.code); //TODO: why red?
```

Comment:
very difficult code construction, thil line has 9 tabs before the code, can we simplify this method?